src/components/map/Map.tsx

```diff
@@ -157,131 +144,264 @@ export const Map = ({
+                          backgroundColor: isActiveMarker
+                            ? marker.iconColor
+                            : marker.iconBackgroundColor
+                        }
+                      ]}
+                    >
+                      <MapIcon
+                        iconColor={
+                          marker.iconColor
+                            ? isActiveMarker
+                              ? colors.surface
+                              : marker.iconColor
+                            : colors.surface
+                        }
+                        iconName={marker.iconName}
+                        iconSize={MARKER_ICON_SIZE / 3.25}
+                      />
+                    </View>
+                  </>
+                ) : (
+                  <MapIcon
+                    iconColor={isActiveMarker ? colors.accent : undefined}
+                    iconName={marker.iconName ? marker.iconName : undefined}
+                  />
+                )}
+                {calloutTextEnabled && (
+                  <View style={styles.callout}>
+                    {!!serviceName && (
+                      <RegularText smallest center>
+                        {serviceName}
+                      </RegularText>
+                    )}
+                    {!!title && (
+                      <RegularText smallest center>
+                        {title}
+                      </RegularText>
+                    )}
                   </View>
-                </>
-              ) : (
-                <MapIcon
-                  iconColor={isActiveMarker ? colors.accent : undefined}
-                  iconName={marker.iconName ? marker.iconName : undefined}
-                />
-              )}
-
-              {calloutTextEnabled && (
-                <Callout style={styles.callout}>
-                  {!!serviceName && (
-                    <RegularText smallest center>
-                      {serviceName}
-                    </RegularText>
-                  )}
-                  {!!title && (
-                    <RegularText smallest center>
-                      {title}
-                    </RegularText>
-                  )}
-                </Callout>
-              )}
-            </Marker>
+                )}
+              </View>
+            </MapLibreGL.PointAnnotation>
           );
         })}
-      </MapView>
+        {!!geometryTourData?.length && (
+          <MapLibreGL.ShapeSource
+            id="line1"
+            shape={{
+              type: 'Feature',
+              geometry: {
+                type: 'LineString',
+                coordinates: geometryTourData.map((point) => [point.longitude, point.latitude])
+              }
+            }}
+          >
+            <MapLibreGL.LineLayer
+              id="linelayer1"
+              style={{ lineColor: colors.primary, lineWidth: 2 }}
+            />
+          </MapLibreGL.ShapeSource>
+        )}
```

Comment:
can you share a screenshot of this as well please

Reply:
![Simulator Screenshot - iPhone 15 Pro - 2024-07-12 at 13 38 23](https://github.com/user-attachments/assets/0d3bcccb-d450-4641-a9a8-67a54291f249)
testing wise I've hardcoded some random coordinates and ist works properly displaying a route.

Reply:
please reduce the size of the screenshot next time or per html image integration here directly. it is hard to have a look with this too big image in the discussions.